src/app/app.component.spec.ts

```diff
@@ -1,18 +1,12 @@
-import { HttpClientModule } from '@angular/common/http';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { TestBed } from '@angular/core/testing';
-import { BrowserModule } from '@angular/platform-browser';
-import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { GatewayService } from './services/gateway.service';
-import { PostService } from './services/post.service';
-import { SubredditService } from './services/subreddit.service';
 
 describe('AppComponent', () => {
     beforeEach(async () => {
         await TestBed.configureTestingModule({
-            imports: [BrowserModule, AppRoutingModule, HttpClientModule],
-            providers: [PostService, SubredditService, GatewayService],
             declarations: [AppComponent],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Why is this necessary in here?

Reply:
It is not I already deleted and fixed problem with adding missing components.